Library/Formula/libiconv.rb

```diff
@@ -22,8 +15,6 @@ class Libiconv < Formula
   patch :DATA
 
   def install
-    ENV.deparallelize
```

Comment:
I'm still getting parallel build issues on a G5. Mind putting this back?

Reply:
done.